src/components/input.js

```diff
@@ -319,15 +319,15 @@ export const AutocompleteTextInput = withAutocomplete(TextInput)
 
 export const DelayedAutoCompleteInput = withDebouncedChange(AutocompleteTextInput)
 
-export const TextArea = ({ onChange, autosize = false, nativeOnChange = false, ...props }) => {
+export const TextArea = Utils.forwardRefWithName('TextArea', ({ onChange, autosize = false, nativeOnChange = false, ...props }, ref) => {
   Utils.useLabelAssert('TextArea', { ...props, allowId: true })
 
   return h(autosize ? TextAreaAutosize : 'textarea', _.merge({
     className: 'focus-style',
     style: styles.textarea,
     onChange: onChange ? (e => onChange(nativeOnChange ? e : e.target.value)) : undefined
-  }, props))
-}
+  }, props), ref)
```

Comment:
Should the ref be in the merged object? I don't think it's doing anything here.

Reply:
This allows us to attach the ref in line 296, otherwise, React complains about the lack of fowardRef

Reply:
I believe Isaac is correct on this. The ref needs to be added to the props otherwise it will not be set correctly:

```javascript  
return h(autosize ? TextAreaAutosize : 'textarea', {
    ..._.merge({
      className: 'focus-style',
      style: styles.textarea,
      onChange: onChange ? (e => onChange(nativeOnChange ? e : e.target.value)) : undefined
    }, props),
    ref
  })
```

You can take a look at [line 84](https://github.com/DataBiosphere/terra-ui/blob/0a2b86be12cae5e22783fae8714dc2925ea3ebc6/src/components/input.js#L84) for an example. You can also test this code out by adding a logging statement around line 243 to display the `current` value and visit the workflow page. Coded as-is, the ref does not get populated.